datatransfer/tasks.py

```diff
@@ -177,10 +209,16 @@ def process_files(source=settings.INGEST_SOURCE_PATH,
         try:
             contents = read_storage.read_file(file_name)
             write_storage.write_file(file_name, contents)
```

Comment:
This probably shouldn't be here. The purpose of the event generation is that the app puts the filename onto a Queue so that another version of DT can consume it. The other running version should be the one to move it.